service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationValueQuantityMapper.java

```diff
@@ -21,21 +21,23 @@
     private static final String GREATER_OR_EQUAL_COMPARATOR_VALUE_TEMPLATE = "<value xsi:type=\"IVL_PQ\"><low value=\"%s\" "
         + "unit=\"%s\" inclusive=\"true\"/></value>";
     private static final String NO_COMPARATOR_NO_SYSTEM_VALUE_TEMPLATE = "<value xsi:type=\"PQ\" value=\"%s\" unit=\"1\">"
-        + "<translation value=\"%s\"><originalText>%s</originalText></translation></value>";
+        + "<translation value=\"%s\">%s</translation></value>";
     private static final String LESS_COMPARATOR_NO_SYSTEM_VALUE_TEMPLATE = "<value xsi:type=\"IVL_PQ\">"
         + "<high value=\"%s\" unit=\"1\" inclusive=\"false\"><translation value=\"%s\">"
-        + "<originalText>%s</originalText></translation></high></value>";
+        + "%s</translation></high></value>";
     private static final String LESS_OR_EQUAL_COMPARATOR_NO_SYSTEM_VALUE_TEMPLATE = "<value xsi:type=\"IVL_PQ\"><high "
         + "value=\"%s\" unit=\"1\" inclusive=\"true\"><translation value=\"%s\">"
-        + "<originalText>%s</originalText></translation></high></value>";
+        + "%s</translation></high></value>";
     private static final String GREATER_COMPARATOR_NO_SYSTEM_VALUE_TEMPLATE = "<value xsi:type=\"IVL_PQ\">"
-        + "<low value=\"%s\" unit=\"1\" inclusive=\"false\"><translation value=\"%s\"><originalText>%s</originalText>"
+        + "<low value=\"%s\" unit=\"1\" inclusive=\"false\"><translation value=\"%s\">%s"
         + "</translation></low></value>";
     private static final String GREATER_OR_EQUAL_COMPARATOR_NO_SYSTEM_VALUE_TEMPLATE = "<value xsi:type=\"IVL_PQ\">"
         + "<low value=\"%s\" unit=\"1\" inclusive=\"true\"><translation value=\"%s\">"
-        + "<originalText>%s</originalText></translation></low></value>";
+        + "%s</translation></low></value>";
     private static final String UNCERTAINTY_CODE = "<uncertaintyCode code=\"U\" "
         + "codeSystem=\"2.16.840.1.113883.5.1053\" displayName=\"Recorded as uncertain\"/>";
+    private static final String QUANTITY_UNIT = "<originalText>%s</originalText>";
```

Comment:
I'd rename it to `QUANTITY_UNIT_TEMPLATE`, to match other format strings. Please let me know.

Reply:
I'm happy with either, if you'd like to update